src/Toml/Parser.hs

```diff
@@ -159,10 +161,19 @@ tableNameP = lexeme $ between (char '[') (char ']') keyP
 
 -- Values
 
+decimalP :: Parser Integer
```

Comment:
I think it's a good idea for you to open an issue in the `megaparsec` library and ask to add some default parser for integers with underscores. You can link `megaparsec` maintainer to your implementation under this PR as an example of such parser 🙂 I think it's a good idea to have such parser in standard parser-combinators library. Especially in the context that there's support for `_` in numeric literals in Haskell since GHC 8.6.1 (enabled by `NumericUnderscores` language extension). And if such parser appears in `megaparsec`, we can replace our parser in `tomland` with more efficient and standard one. I think this will happen only about one month in `megaparec` library and we still can have this feature in `tomland`. 

I'm okay if the parser in `megaparsec` doesn't parse things like `13_`, it's still better to use standard function, especially then the specification of configuration format is not precise enough.

Reply:
Great idea. I'll open a PR for megaparsec and come back and update tomland if it's accepted 👍